### ISSUE TITLE
How can I get the score of each class?

### ISSUE THREAD

**Issue by Zhengym-hub** [jindongwang/Pytorch-CapsuleNet#15]
Hello, I have run this model in myself dataset, but I find that it can only get three parameters: output, reconstructions and masked. I want to calculate the AUC, it needs score of each class, so, how can I get the parameter? Thanks.

**Comment by jindongwang** [comment]
I think it is not easy to get the logits from capsules... This is different from softmax where we can get the logits. Maybe it is not suitable for your task.

**Comment by Zhengym-hub** [comment]
> I think it is not easy to get the logits from capsules... This is different from softmax where we can get the logits. Maybe it is not suitable for your task.

Ok, thank you.
